### ISSUE TITLE
Crash on Android - Can only use lower 16 bits for requestCode

### ISSUE THREAD

**Issue by gorjuspixels** [tipsi/tipsi-stripe#296]
The app crashes on Android with the following stack trace:

```
java.lang.IllegalArgumentException: Can only use lower 16 bits for requestCode
    at android.support.v4.app.BaseFragmentActivityApi14.checkForValidRequestCode(BaseFragmentActivityApi14.java:79)
    at android.support.v4.app.BaseFragmentActivityApi14.startIntentSenderForResult(BaseFragmentActivityApi14.java:66)
    at android.support.v4.app.FragmentActivity.startIntentSenderForResult(FragmentActivity.java:67)
    at com.google.android.gms.common.api.Status.startResolutionForResult
    at com.google.android.gms.common.api.ResolvableApiException.startResolutionForResult
    at com.google.android.gms.wallet.AutoResolveHelper.zza
    at com.google.android.gms.wallet.AutoResolveHelper.zzb
    at com.google.android.gms.wallet.AutoResolveHelper$zzb.zze
    at com.google.android.gms.wallet.AutoResolveHelper$zzb.zza
    at com.google.android.gms.wallet.AutoResolveHelper$zza.zzbjy
    at com.google.android.gms.wallet.AutoResolveHelper$zza.onComplete
    at com.google.android.gms.tasks.zzf.run
    at android.os.Handler.handleCallback(Handler.java:790)
    at android.os.Handler.dispatchMessage(Handler.java:99)
    at android.os.Looper.loop(Looper.java:164)
    at android.app.ActivityThread.main(ActivityThread.java:6494)
    at java.lang.reflect.Method.invoke(Method.java)
    at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)
```

FragmentActivity requires the requestCode to be of 16 bits, meaning the range is from 0 to 65535. So when calling `AutoResolveHelper.resolveTask`, the requestCode passed to it must be between 0 and 65535.. but it's 100500. It works if i change the value to anything less than or equal to 65535.

EDIT: the constant that I'm talking about is `LOAD_PAYMENT_DATA_REQUEST_CODE`

**Comment by isnifer** [comment]
@gorjuspixels Hi! Can you describe your case? I don't understand

**Comment by brilliantjc** [comment]
How and where to change?^_^

**Comment by brilliantjc** [comment]
I see
<img width="748" alt="screen shot 2018-05-31 at 3 18 59 pm" src="https://user-images.githubusercontent.com/25623417/40811421-fba04882-64e5-11e8-9de8-9908dad759d1.png">

**Comment by brilliantjc** [comment]
PR done, thank you for all your support, isnifer!

**Comment by brilliantjc** [comment]
Thank you so much!!! @isnifer

**Comment by isnifer** [comment]
Fixed in 5.2.3

**Comment by gorjuspixels** [comment]
Great! Thank you <3
